js/app/i18n/locales/en.json

```diff
@@ -93,6 +93,7 @@
         "ADMIN_DASHBOARD_FILTERS_SHOW_ALL": "Show all",
         "ADMIN_DASHBOARD_FILTERS_HIDE": "Hide",
         "ADMIN_DASHBOARD_HIDE_SHOW_ON_MAP": "Hide/show on map",
+        "ADMIN_DASHBOARD_ONLY_UNASSIGNED_WITH_SHIFT": "Only unassigned tasks column handle shift selection",
```

Comment:
we can mention that a user can use `command` key on macos / `Ctrl`? on windows to select individual tasks. I tested it works for both unassigned and assigned tasks

Reply:
Yes! With screenshots. Shift, for example, did not even know that the right panel was resizable